### DISCUSSION TITLE
Where is my per day activity?

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [community/community#27569] [Other Feature Feedback, Questions, & Ideas]
I want to see what I did on a specific day.

**Comment by radar-blip** [comment]
Activity is located at the bottom of your main page, below the chart thingy.

**Reply by vitaly-zdanevich** [reply]
![image](https://user-images.githubusercontent.com/3514015/181701513-174019ba-1c93-41aa-be06-7b25c1887d6a.png)

What I did yesterday? There is no such info.

**Reply by radar-blip** [reply]
Try clicking on commits, just as an experiment.

**Reply by radar-blip** [reply]
Mine shows commits up until 2 hours ago, which is correct.

**Reply by vitaly-zdanevich** [reply]
> Try clicking on commits

I see commits for a month :(

**Comment by jfullstackdev** [comment]
what activity you are referring too ? because if that is about commits on a repo, your last one was forking 3 days ago

![image](https://user-images.githubusercontent.com/47092464/181702270-129de14c-ffaa-4eac-9097-f4b2dd367002.png)

your last commits on a repo was 17 days ago 

![image](https://user-images.githubusercontent.com/47092464/181702515-677fab2f-9268-4ec7-8ba4-f2f48bf5bf9d.png)

but your most recent ones are posting to discussions and one issue

![image](https://user-images.githubusercontent.com/47092464/181702660-7f671eea-2b3f-4840-a40d-1de739877a02.png)

**Reply by vitaly-zdanevich** [reply]
I want to see all my commits and discussions what were yesterday, for example, but I have only for a month:
![image](https://user-images.githubusercontent.com/3514015/181703547-7cfb1d52-915c-408f-959b-7bb70fcba4d5.png)

**Reply by jfullstackdev** [reply]
if you click those green tiles one by one, the activity at the bottom will reveal what are those, same thing I did to enumerate all of your activities

![image](https://user-images.githubusercontent.com/47092464/181705159-5c945c9b-a709-4650-a90a-db3f319834dc.png)

**Reply by vitaly-zdanevich** [reply]
YES THANK YOU this is the answer!
